tests/integration/test_orders_cli.py

```diff
@@ -65,7 +65,11 @@ def test_cli_orders_list_basic(invoke, order_descriptions):
 
     result = invoke(['list'])
     assert not result.exception
-    assert [order1, order2, order3] == json.loads(result.output)
```

Comment:
The decoupling I've done means that CLI output is no longer exactly the same as the API response.

Specifically: 

* Order properties other than the ones specified in the class definition are gone from the output JSON. They were already hidden by the Python API (only id and state were surfaced).
* `_links.results` has been flattened to `results`.